endpoints/tweet.py

```diff
@@ -39,7 +39,7 @@ def generate(self, avatars, text, usernames):
         tag_raw = usernames[1] if len(usernames) == 2 else usernames[0]
         text3 = wrap(font3, f'@{tag_raw}', 1150)
 
-        time = datetime.now().strftime('%-I:%M %p - %d %b %Y')

```

Comment:
Krom and I already talked about this. Keep the '%-I', it makes it 12H which is more consistent with Twitter

Reply:
didn't i revert this? Back to your version